<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>624</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_english_app">
     <property name="font">
      <font>
       <family>Segoe UI,Inter,sans-serif</family>
       <pointsize>-1</pointsize>
       <weight>99</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Uygulama Başlık Etiketi - lbl_english_app */
QLabel#lbl_english_app {
    /* Yazı Rengi: Siyah yerine derin ve zengin bir indigo mavi */
    color: #312E81; 
    
    /* Font Ayarları: Modern, kalın ve okunaklı */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, sans-serif;
    font-size: 32px;
    font-weight: 800; /* Ekstra kalın (Extra Bold) */
    
    /* Arka plan şeffaf kalsın */
    background-color: transparent;
    
    /* Harf boşluğunu biraz açarak daha premium bir hava katalım */
    letter-spacing: 1px;
    
    /* Etrafında biraz boşluk bırakalım */
    padding: 10px;
}

/* Eğer başlığın altına çok hafif bir renkli gölge istersen (isteğe bağlı) */
/* Not: Bu sadece yazıya odak katmak içindir */
QLabel#lbl_english_app:hover {
    color: #4338CA; /* Fareyle üzerine gelince (eğer etkileşimliyse) hafifçe açılır */
}</string>
     </property>
     <property name="text">
      <string>My English App</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="line_user_name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #C7D2FE;
    border-radius: 12px;
    padding: 12px;
    background-color: white;
    color: #3730A3;
    font-size: 14px;
}
QLineEdit:focus {
    border: 2px solid #818CF8; /* Odaklanınca renk canlanır */
    background-color: #FDFDFF;
}</string>
        </property>
        <property name="placeholderText">
         <string>Kullanıcı adınız</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_login">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI,sans-serif</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Ana Giriş Butonu - btn_login */
QPushButton#btn_login {
    /* Soft ama etkileyici bir mavi-mor geçişi */
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
                stop:0 #6366F1, stop:1 #8B5CF6);
    
    /* Yazı bembeyaz ve net */
    color: #FFFFFF; 
    
    /* Kenarlık siyah değil, butonun kendi renginde çok ince bir hat */
    border: 1px solid #4F46E5;
    
    border-radius: 12px;
    padding: 12px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 15px;
    font-weight: bold;
}

/* Fare üzerine geldiğinde renkler canlanır */
QPushButton#btn_login:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
                stop:0 #4F46E5, stop:1 #7C3AED);
    border: 1px solid #4338CA;
}

/* Tıklandığında derinlik hissi */
QPushButton#btn_login:pressed {
    background-color: #4338CA;
    /* Tıklandığında hafifçe aşağı çökmüş gibi görünmesi için */
    padding-top: 14px;
    padding-bottom: 10px;
}</string>
        </property>
        <property name="text">
         <string>Giriş yap</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_forgot_password">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Şifremi Unuttum Butonu - btn_forgot_password */
QPushButton#btn_forgot_password {
    /* Arka plan tamamen şeffaf, sadece yazı görünsün */
    background-color: transparent;
    
    /* Yazı rengi: Siyah yerine soft bir gri-mavi (Modern ton) */
    color: #64748B; 
    
    /* Kenarlık (Siyah çizgi) tamamen kaldırıldı */
    border: none;
    
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 12px;
    font-weight: 500;
    padding: 5px;
}

/* Fare üzerine geldiğinde (Vurgu efekti) */
QPushButton#btn_forgot_password:hover {
    /* Yazı rengi ana temanın rengine (Mavi/Mor) döner */
    color: #6366F1;
    /* Altı çizili yaparak link olduğu hissini güçlendirir */
    text-decoration: underline;
}

/* Tıklandığında */
QPushButton#btn_forgot_password:pressed {
    color: #4338CA;
}</string>
        </property>
        <property name="text">
         <string>Şifremi unuttum</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_user_name">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #3730A3; /* Koyu soft indigo - Siyah gibi sert durmaz */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;
    font-weight: 600; /* Yazıyı biraz daha okunaklı yapar */
    background-color: transparent; /* Arka planı etkilemesin */
}</string>
        </property>
        <property name="text">
         <string>Kullanıcı Adı</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_password">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #3730A3; /* Koyu soft indigo - Siyah gibi sert durmaz */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;
    font-weight: 600; /* Yazıyı biraz daha okunaklı yapar */
    background-color: transparent; /* Arka planı etkilemesin */
}</string>
        </property>
        <property name="text">
         <string>Şifre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_password">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #C7D2FE;
    border-radius: 12px;
    padding: 12px;
    background-color: white;
    color: #3730A3;
    font-size: 14px;
} 
QLineEdit:focus {
    border: 2px solid #818CF8; /* Odaklanınca renk canlanır */
    background-color: #FDFDFF;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Şifrenizi girin</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="lbl_signup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #3730A3; /* Koyu soft indigo - Siyah gibi sert durmaz */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;
    font-weight: 600; /* Yazıyı biraz daha okunaklı yapar */
    background-color: transparent; /* Arka planı etkilemesin */
}</string>
        </property>
        <property name="text">
         <string>Hesabınız yok mu? Buradan kayıt olabilirsiniz.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btn_signup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Kayıt Ol Butonu - btn_signup */
QPushButton#btn_signup {
    /* Arka plan: Çok yumuşak bir pastel lila/mavi */
    background-color: #EEF2FF; 
    
    /* Yazı rengi: Siyah yerine modern bir indigo */
    color: #4F46E5; 
    
    /* Kenarlık: Siyah yerine çok ince ve şeffaf bir mor-mavi hat */
    border: 1px solid rgba(79, 70, 229, 0.2); 
    
    border-radius: 12px;
    padding: 10px 20px;
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, sans-serif;
    font-size: 13px;
    font-weight: 600;
}

/* Fare üzerine geldiğinde (Soft geçiş) */
QPushButton#btn_signup:hover {
    background-color: #E0E7FF; /* Biraz daha belirgin bir mavi-mor */
    color: #4338CA;
    border: 1px solid #C7D2FE;
}

/* Tıklandığında (Tıklama hissi) */
QPushButton#btn_signup:pressed {
    background-color: #C7D2FE;
    border: none;
}</string>
        </property>
        <property name="text">
         <string>Kayıt ol</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>501</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
